### PR TITLE
deployment event

### PR DESCRIPTION
**Pull request by mattsonjoshcf**
- [X] Mini bio
Senior Application Support Engineer and 6 year Racker on the Critical Application Services team. Specializing in Java web applications
 - [X] Desired date of publishing
11/16/16
 - [X] Social media contact info
N/A
 - [X] Redaction: (yes|no)
Yes
 - [X] Comments: (yes|no)
No

@catlook Please review.

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-e29687591b/).

File: _posts/2016-11-16-Using-AppDynamics-to-Track-Performance-Between-Releases.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+<!-- more -->
+
+
+Fortunately, AppDynamics includes two features that can easily allow us to automatically record a deployment event within AppDynamics and use that as a reference for comparing releases.
+
+### Deployment Markers
+
+AppDynamics events can be triggered for a variety of reasons including health rule violations, application state changes (like a restart) and others; however, the API also allows us to create [custom events](https://docs.appdynamics.com/display/PRO42/Alert+and+Respond+API#AlertandRespondAPI-CreateaCustomEvent).  By taking advantage of this feature, we can modify our deployment process to include a REST API call to trigger a custom event and programmatically record deployments within AppDynamics itself.
+
+The process is simple enough; by using cURL we can plug in the appropriate variables and create our API call.  For example, we might use something like this:
```
Comment (catlook) [inline _posts/2016-11-16-Using-AppDynamics-to-Track-Performance-Between-Releases.md]: ...  enough. We can use cURL to plug in...

File: _posts/2016-11-16-Using-AppDynamics-to-Track-Performance-Between-Releases.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+In this example, I've hard-coded the release number, but it should be trivial to include your release number as a variable here for greater flexibility.  After executing this API call, we can now see the event within the AppDynamics console:
+
+![Deployment Event]({% asset_path 2016-11-16-Using-AppDynamics-to-Track-Performance-Between-Releases/deployment_event.png %})
+
+### Release Comparison
+
+Now that we have our deployments automatically being recorded within AppDynamics as an event, we can use these timestamps as a starting point for later analysis.  This is where the [Compare Releases](https://docs.appdynamics.com/display/PRO42/Metrics+and+Graphs) function of AppDynamics comes in handy.
+
+The Compare Releases functionality is straight forward; all we have to do is pick a time frame before the recorded release time and after the recorded release time to compare.  AppDynamics will allow us to compare performance in broad and granular capacities.
```
Comment (catlook) [inline _posts/2016-11-16-Using-AppDynamics-to-Track-Performance-Between-Releases.md]: ... forward. All we ...
.. release time and pick one after the ....

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-8bb2dc8a6f/).

**Comment by mattsonjoshcf** [comment]
Thanks, Cat!  Should be all good to go now.

**Comment by catlook** [comment]
Golden! Thanks, Josh!

**Pull request merged**